app.js

```diff
@@ -7,6 +7,13 @@ const govukTemplate = require('hof-govuk-template');
 const hoganExpressStrict = require('hogan-express-strict');
 const expressPartialTemplates = require('express-partial-templates');
 const _ = require('lodash');
+const apiCall = require('./lib/api-call');
+const config = require('./config');
+const baseUrl = config.apiBaseUrl;
```

Comment:
@daniel-ac-martin 's suggestion I don't think has a special name(?) its definitely not destructuring